blueprints/secops/secops-gke-forwarder/main.tf

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  fleet_host = join("", [
+    "https://connectgateway.googleapis.com/v1/",
+    "projects/${module.project.number}/",
+    "locations/global/gkeMemberships/${var.chronicle_forwarder.cluster_name}"
+  ])
+}
+
+module "project" {
+  source = "./../../../modules/project"
+  billing_account = (var.project_create != null
+    ? var.project_create.billing_account_id
+    : null
+  )
+  parent = (var.project_create != null
+    ? var.project_create.parent
+    : null
+  )
+  prefix         = var.project_create == null ? null : var.prefix
```

Comment:
I reckon `prefix` isn't used if project_create == false, so you can probably simplify this.

Reply:
Simplified!